src/utils/firebase-db.ts

```diff
@@ -33,156 +29,140 @@ const app = initializeApp(firebaseConfig);
 // Initialize firebase database and get the reference of firebase database object.
 const database = getDatabase(app);
 
-// I running npm run dev, the application will try and connect to the local firebase real time database emulator
+// if running npm run dev, the application will try and connect to the local firebase real time database emulator
+// Does not seem to be working
 // https://firebase.google.com/docs/emulator-suite/install_and_configure
-if (location.hostname === "localhost" && import.meta.env.VITE_MODE === "dev") {
```

Comment:
I would leave this in to be able to connect to the emulator.

You have to start  and seed the emulator before it works. If we take this out we will be doing development against a production database which is not a best practice

Reply:
I tried with or without it but it was not working, I was still connected to the online realtime database it seems, but I will but this code snippet.